### PR TITLE
Devise token auth

### PR DESCRIPTION
**Pull request by alanlamaswolox**
## Summary
Devise-token-auth installed and configured

## Trello
https://trello.com/c/55OYuxCH/25-10-implementar-autenticaci%C3%B3n

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,4 +1,13 @@
 # Default controller created by Rails
 class ApplicationController < ActionController::Base
+  include DeviseTokenAuth::Concerns::SetUserByToken
   protect_from_forgery with: :exception
```
Comment (nicozar95) [inline app/controllers/application_controller.rb]: Remove the `:exception`

**Review by nicozar95 — CHANGES_REQUESTED** [review]
Change the protect_forgery

File: config/environments/development.rb [hunk 2]
```diff
@@ -53,4 +53,6 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
```
Comment (nicozar95) [inline config/environments/development.rb]: Does'nt rubocop complain about this extra line?

Comment (alanlamaswolox) [reply config/environments/development.rb]: No, rubocop inspects 9 files and says no offenses

Comment (nicozar95) [reply config/environments/development.rb]: Ok, then delete the extra line :sweat_smile: please!!! :pray:

Comment (fedeest) [reply config/environments/development.rb]: Rubocop doesn't check the files under config

**Review by fedeest — APPROVED** [review]

**Review by nicozar95 — APPROVED** [review]

**Pull request merged**